src/styles/buttons.js

```diff
@@ -0,0 +1,31 @@
+import makeStyles from "@material-ui/core/styles/makeStyles";
+import colors from "./colors";
+
+export default {
+  RetryUpload: makeStyles(theme => ({
+    button: {
+      marginTop: theme.spacing(1),
+      marginRight: theme.spacing(1),
+      backgroundColor: colors.presqtBlue,
+      "&:hover": {
+        backgroundColor: "#0a4996"
```

Comment:
Should we put this in the color styling? presqtBlueHover?

Reply:
Yep